### PR TITLE
docs: Add QA about exposing gateway as loadbalancer

### PR DESCRIPTION
**Pull request by Revolyssup**
### Type of change:
Solves #1895 

- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches
- [x] Documentation
- [ ] Refactor
- [ ] Chore
- [ ] CI/CD or Tests

### What this PR does / why we need it:
This PR adds answer to a commonly asked question by users.

### Pre-submission checklist:

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [x] Have you modified the corresponding document?
- [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -88,3 +88,12 @@ You can change the Admin API key in two ways:
 
 1. Modify the key in both [apisix/values.yaml](https://github.com/apache/apisix-helm-chart/blob/57cdbe461765cd49af2195cc6a1976cc55262e9b/charts/apisix/values.yaml#L181) and [apisix/apisix-ingress-controller/values.yaml](https://github.com/apache/apisix-helm-chart/blob/57cdbe461765cd49af2195cc6a1976cc55262e9b/charts/apisix-ingress-controller/values.yaml#L128) files.
 2. You can also set this imperatively by passing the flag `--set ingress-controller.config.apisix.adminKey=<new key> --set admin.credentials.admin=<new key>` to the `helm install` command.
+
+
+## Why does my Ingress resource not have an address?
+
+You will need to get the apisix-gateway service an external IP assigned for it to reflect on the Ingress's status.
+1. While installing APISIX helm chart make sure to override gateway type with `--set gateway.type=LoadBalancer`. 
+2. Also make sure to pass ingressPublishService while installing Ingress controller with `--set ingress-controller.config.ingressPublishService=apisix-gateway`
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:97]: The format of "service" is ns/service.

File: docs/en/latest/FAQ.md [hunk 2]
```diff
@@ -88,3 +88,12 @@ You can change the Admin API key in two ways:
 
 1. Modify the key in both [apisix/values.yaml](https://github.com/apache/apisix-helm-chart/blob/57cdbe461765cd49af2195cc6a1976cc55262e9b/charts/apisix/values.yaml#L181) and [apisix/apisix-ingress-controller/values.yaml](https://github.com/apache/apisix-helm-chart/blob/57cdbe461765cd49af2195cc6a1976cc55262e9b/charts/apisix-ingress-controller/values.yaml#L128) files.
 2. You can also set this imperatively by passing the flag `--set ingress-controller.config.apisix.adminKey=<new key> --set admin.credentials.admin=<new key>` to the `helm install` command.
+
+
+## Why does my Ingress resource not have an address?
+
+You will need to get the apisix-gateway service an external IP assigned for it to reflect on the Ingress's status.
+1. While installing APISIX helm chart make sure to override gateway type with `--set gateway.type=LoadBalancer`.
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:96]: Setting it as NodePort will also work.

Comment (Revolyssup) [reply docs/en/latest/FAQ.md:96]: But in case of NodePort, an external IP will not be allocated to it, right?

Comment (tao12345666333) [reply docs/en/latest/FAQ.md:96]: yes.

https://github.com/apache/apisix-ingress-controller/blob/e809cfbaaad6f6c3cbd23f5ec8e93bc1d87f543d/conf/config-default.yaml#L44

we can use `ingress_status_address`

see the comment:

>   when there is no available information on the Service
> used for publishing on the data plane,
> the static address provided here will be
>  used to update the status information of Ingress.
>  When ingress-publish-service is specified at the same time, ingress-status-address is preferred.
> For example, no available LB exists in the bare metal environment.

Comment (Revolyssup) [reply docs/en/latest/FAQ.md]: Applied suggestion

File: docs/en/latest/FAQ.md (hunk 1 above)
Comment (Revolyssup) [reply docs/en/latest/FAQ.md]: Applied suggestion

**Review by Sn0rt — APPROVED** [review]

**Review by leslie-tsang — APPROVED** [review]

**Pull request merged**